packages/@o3r/core/project.json

```diff
@@ -14,6 +14,7 @@
         "script": "postbuild"
       },
       "dependsOn": [
+        "^build",
```

Comment:
It is supposed to be the job of the `compile` task

Reply:
indeed, this was unnecessary